plenum/test/test_node.py

```diff
@@ -379,7 +379,7 @@ def __init__(self,
         self.pluginPaths = pluginPaths
 
         self.testNodeClass = testNodeClass
-        self.nodes = OrderedDict()  # type: Dict[str, TestNode]
+        self.nodes = SortedDict()  # type: Dict[str, TestNode]
```

Comment:
This change was done to fix tests logic. We expect nodes to be in following order: 
`Alpha, Beta, Gamma, Delta`
as they are in Greek alphabet, but nodes sort their remotes by name during primary selection and the order changes: 
`Alpha, Beta, Delta, Gamma`.

This causes following problems:

1. If you want to stop non primary you stop last one, which you expect to be Delta - but it is actually Gamma. And if you then start a view change you get unexpected behaviour - instance 1 decides Delta to be next primary, but Delta is down.

2. It confuses you when you read test logs